scripts/data_ingest/main.py

```diff
@@ -0,0 +1,147 @@
+import logging
+import logging.config
+import os
+import sys
+from http import HTTPStatus
+from pathlib import Path
+from pprint import pprint
+
+import requests
+
+ADMIN_EMAIL_ENV = "SUPERUSER_EMAIL"
+ADMIN_PASSWORD_ENV = "SUPERUSER_PASSWORD"
+ADMIN_TOKEN_ENV = "SUPERUSER_TOKEN"
+BULK_IMPORT_ENDPOINT = "/api/v1/bulk-imports/cclw/law-policy"
+
+
+DEFAULT_LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "standard": {"format": "%(asctime)s [%(levelname)s] %(name)s: %(message)s"},
+    },
+    "handlers": {
+        "default": {
+            "level": "INFO",
+            "formatter": "standard",
+            "class": "logging.StreamHandler",
+            "stream": "ext://sys.stdout",  # Default is stderr
+        },
+    },
+    "loggers": {
+        "": {  # root logger
+            "handlers": ["default"],
+            "level": "INFO",
+        },
+        "__main__": {  # if __name__ == '__main__'
+            "handlers": ["default"],
+            "level": "DEBUG",
+            "propagate": False,
+        },
+    },
+}
+
+logging.config.dictConfig(DEFAULT_LOGGING)
+logger = logging.getLogger(__file__)
+
+
+def _log_response(response: requests.Response) -> None:
+    if response.status_code >= 400:
+        logger.error(
+            f"There was an error during a request to {response.url}. "
+            f"STATUS: {response.status_code}, BODY:{response.content!r}"
```

Comment:
extra: {props: {}} here?

Reply:
this is just a temporary admin script for running locally until we have an admin interface, not something to be logging into CloudWatch, so I think this is okay